Ladybird/AudioCodecPluginLadybird.cpp

```diff
@@ -68,6 +68,7 @@ class AudioThread final : public QThread { // We have to use QThread, otherwise
 
 Q_SIGNALS:
     void playback_position_updated(Duration);
+    void seek_completed();
```

Comment:
I think your squashes got mixed up, these seek completion changes should be in the previous commit